tests/backends/base/test_creation.py

```diff
@@ -188,3 +188,45 @@ def test_serialize_db_to_string_base_manager(self):
             data = connection.creation.serialize_db_to_string()
         self.assertIn('"model": "backends.schoolclass"', data)
         self.assertIn('"year": 1000', data)
+
```

Comment:
I'm not sure if it's the best place for these tests :thinking:, maybe `backends/base/test_base.py`.

Reply:
It's here because `mark_expected_failures_and_skips()` is on `DatabaseCreation` (`creation.py`). `test_base.py` is for `DatabaseWrapper` (`base.py`).